cirq/vis/heatmap.py

```diff
@@ -52,220 +35,318 @@ def relative_luminance(color: np.ndarray) -> float:
     return vis_utils.relative_luminance(color)
 
 
-class Heatmap:
-    """Distribution of a value in 2D qubit lattice as a color map."""
+QubitTuple = Tuple[grid_qubit.GridQubit, ...]
+Point = NamedTuple('Point', [('x', SupportsFloat), ('y', SupportsFloat)])
+PolygonUnit = NamedTuple(
+    'PolygonUnit',
+    [('polygon', Polygon), ('center', Point), ('value', SupportsFloat), ('annot', str)],
+)
 
-    def __init__(self, value_map: ValueMap) -> None:
-        self.set_value_map(value_map)
-        self.annot_map = {  # Default annotation.
-            _get_qubit_row_col(qubit): format(float(value), '.2g')
-            for qubit, value in value_map.items()
-        }
-        self.annot_kwargs: Dict[str, Any] = {}
-        self.unset_url_map()
-        self.set_colorbar()
-        self.set_colormap()
 
-    def set_value_map(self, value_map: ValueMap) -> 'Heatmap':
-        """Sets the values for each qubit.
+class Heatmap:
+    """Distribution of a value in 2D qubit lattice as a color map."""
 
-        Args:
-            value_map: the values for determining color for each cell.
+    def __init__(self, value_map: Mapping[QubitTuple, SupportsFloat], **kwargs):
+        """2D qubit grid Heatmaps
+
+        Draw 2D qubit grid heatmap with Matplotlib with parameters to configure the properties of
+        the plot.
+
+        Parameters
```

Comment:
Please follow the [Google style guide regarding docstrings](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods). I.e. Args instead of Parameters

Reply:
done